bash_completion

```diff
@@ -1821,7 +1821,7 @@ _comp_compgen_available_interfaces()
             ip -c=never link show || ip link show || ifconfig -a
         fi
     } 2>/dev/null | _comp_awk \
-        '/^[^ \t]/ { if ($1 ~ /^[0-9]+:/) { print $2 } else { print $1 } }')" &&
+        '/^[^ \t]/ { if ($1 ~ /^[0-9]+:/) { split($2, link, "@"); print link[1] } else { print $1 } }')" &&
```

Comment:
I'd rather use `sub(/@.*/, "", $2); print $2`, or another option is to include `@` as a part of the separators, `-F '[ @]'`.

Reply:
I think I prefer the `sub` option, but no strong preference

Reply:
I believe this is fair statement, providing new revision now.